mureq.py

```diff
@@ -112,25 +159,34 @@ def yield_response(method, url, *, unix_socket=None, timeout=DEFAULT_TIMEOUT, he
     :return: http.client.HTTPResponse, yielded as context manager
     :rtype: http.client.HTTPResponse
     :raises: HTTPException
-    """
+    """  # noqa: E501
     method = method.upper()
-    headers = _prepare_outgoing_headers(headers)
+    headers = cast("MutableMapping[str, str]", _prepare_outgoing_headers(headers))
     enc_params = _prepare_params(params)
-    body = _prepare_body(body, form, json, headers)
+    prepared_body: bytes | str | None = _prepare_body(body, form, json, headers)
```

Comment:
nitpick: is this local annotation necessary?

Reply:
Removed in latest commit